index.html

```diff
@@ -189,7 +200,33 @@ <h3><img src="images/circle.png" alt="">Skills</h3>
 				<li>Terminal</li>
 			</ul>
 		</div>
+	</div>
 	</section>
 
+	<section class="contact-me">
+		<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+		<div class="last">
+		<form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] kindly consider going to [formspree](https://formspree.io/html/) and creating a form and integrating it into your form this will make your form successfully submitted when the user enters the required data
- cause, when I submit it here, is what I am getting 👇 
![by](https://user-images.githubusercontent.com/60146030/221941310-f008a22b-bdd7-4273-9c3f-aba8c3d2e88a.PNG)